git/diff.go

```diff
@@ -0,0 +1,48 @@
+package git
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+
+	"github.com/rs/zerolog/log"
+	"github.com/waigani/diffparser"
+)
+
+func gitDiff(dir string) (string, error) {
+	var out bytes.Buffer
+	cmd := exec.Command("git", "diff", "-G", "(^[^#])", "--staged")
```

Comment:
@alephnull as implemented right now, the diff command will only show output if files has been already staged, which is not the case for the doctor workflow. I would rather remove '--staged' from the cmd command or make it optional through an optional flag. I will send a PR for the second

Reply:
This is a good point. I have reviewed #113 .